server/apps/distributed-app/sample-configuration/logback.xml

```diff
@@ -11,32 +11,11 @@
                         <immediateFlush>false</immediateFlush>
                 </encoder>
         </appender>
-
-        <appender name="LOG_FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-                <file>/logs/james.log</file>
-                <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-                    <fileNamePattern>/logs/james.%i.log.tar.gz</fileNamePattern>
-                    <minIndex>1</minIndex>
-                    <maxIndex>3</maxIndex>
-                </rollingPolicy>
-
-                <triggeringPolicy class="ch.qos.logback.core.rolling.SizeBasedTriggeringPolicy">
-                    <maxFileSize>100MB</maxFileSize>
-                </triggeringPolicy>
-
-                <encoder>
-                        <pattern>%d{HH:mm:ss.SSS} [%-5level] %logger{15} - %msg%n%rEx</pattern>
-                        <immediateFlush>false</immediateFlush>
-                </encoder>
-        </appender>
-
         <root level="WARN">
                 <appender-ref ref="CONSOLE" />
-                <appender-ref ref="LOG_FILE" />
         </root>
 
         <logger name="org.apache.james" level="INFO" />
         <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
 
-
-</configuration>
+</configuration>
```

Comment:
please don't edit the current logback file
Create one new, then mount it to docker-compose

Reply:
He is just removing the appender that was only  created for crowdsec but not needed anymore? surely we dont want to keep that right? :)

Reply:
@Arsnael  This is logback file in distribute-app, so we should not modify it, right? Similar with memory-app?

Reply:
Maybe we don't want to keep that, but  I think it will be better if has another logback.xml in `crowdsec` module.

Reply:
Ok sorry, I thought this was introduced by this PR but it was already there, @vttranlina comment is valid actually :)